standalone/docusaurus/docs/componenten/Heading.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Heading
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Heading
+sidebar_position: 1
+pagination_label: Heading
+description: Utrecht Heading component.
+keywords:
+  - h1
+  - h2
+  - h3
+  - h4
+  - h5
+  - h6
+  - heading
+  - utrecht-heading
+---
+
+import { Markdown } from "../../src/components/Markdown";
```

Comment:
I have tried to include `<!-- @license CC0-1.0 -->` in every `*.md`/`*.mdx` documentation page, so it easy to copy to other repos while maintaining the open source license and it is clear that the license for docs is different from the code repository overall.